spec/controllers/target_controller_spec.rb

```diff
@@ -39,6 +39,7 @@
       end
 
       context 'JSON format' do
```

Comment:
Try to make the context start with `when`, `with`, or a verb in the continuous form so when you read the whole test scenario it makes sense. 
`GET targets when sending in JSON format when a target has a match its compatible target is returned.`